### PR TITLE
New changelog item

### PR DESCRIPTION
**Pull request by jvillafanez**
## Description
New changelog item

## Related Issue
Linking to https://github.com/owncloud/core/pull/36242

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: changelog/unreleased/36242 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix SMB access denied error while listing the contents of the folder
+
+This can happen in a DFSr folder, where such folder is visible even though the user doesn't have permissions to read the folder.
```
Comment (micbar) [inline changelog/unreleased/36242]: The long description needs to be in past tense.

**Review by micbar — CHANGES_REQUESTED** [review]
Please correct the time.

**Review by micbar — APPROVED** [review]
👍

File: changelog/unreleased/36242 [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix SMB access denied error while listing the contents of the folder
+
+This happened in a DFSr folder, where such folder was visible even though the user didn't have permissions to read the folder.
```
Comment (mmattel) [inline changelog/unreleased/36242]: For better understanding, proposal: `DFSr` --> `DFS Replication (DFSr)`

**Review by phil-davis — APPROVED** [review]

**Pull request merged**